products/extensions/search/Cross-search-logic.md

```diff
@@ -0,0 +1,23 @@
+# Cross-search logic
+
+@See `\Shopware\Commercial\AdvancedSearch\Domain\CrossSearch\CrossSearchLogic`
```

Comment:
What does @See mean? does it mean "refer to" ?

Reply:
@sushmangupta yes it is